programs/drift/src/state/perp_market.rs

```diff
@@ -298,6 +300,10 @@ impl MarketIndexOffset for PerpMarket {
 }
 
 impl PerpMarket {
+    pub fn get_oracle_id(&self) -> OracleIdentifier {
```

Comment:
drop the `get` its cleaner
```suggestion
    pub fn oracle_id(&self) -> OracleIdentifier {
```